testing/test_config.py

```diff
@@ -612,20 +612,14 @@ def pytest_addoption(parser):
         assert config.getini("custom") == "1"
 
     def test_absolute_win32_path(self, pytester: Pytester) -> None:
```

Comment:
I tracked this test to this changelog entry:

```
* Fix win32 path issue when puttinging custom config file with absolute path 
  in ``pytest.main("-c your_absolute_path")``.
```

Seems `--version` being in the config file has nothing to do with the original issue.